src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/QueryHandlers/UpdateApprenticeshipHandler.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Models;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries;
+using Microsoft.Azure.Documents.Client;
+using OneOf.Types;
+
+namespace Dfc.CourseDirectory.Core.DataStore.CosmosDb.QueryHandlers
+{
+    public class UpdateApprenticeshipHandler : ICosmosDbQueryHandler<UpdateApprenticeship, Success>
+    {
+        public async Task<Success> Execute(DocumentClient client,
+            Configuration configuration, UpdateApprenticeship request)
+        {
+            var documentUri = UriFactory.CreateDocumentUri(
+                configuration.DatabaseId,
+                configuration.ApprenticeshipCollectionName,
+                request.Id.ToString());
+
+            var apprenticeship = new Apprenticeship()
+            {
+                Id = request.Id,
+                ProviderUKPRN = request.ProviderUkprn,
+                ApprenticeshipTitle = request.ApprenticeshipTitle,
+                ApprenticeshipType = request.ApprenticeshipType,
+                MarketingInformation = request.MarketingInformation,
+                Url = request.Url,
+                ContactTelephone = request.ContactTelephone,
+                ContactEmail = request.ContactEmail,
+                ContactWebsite = request.ContactWebsite,
+                ApprenticeshipLocations = request.ApprenticeshipLocations.Select(l => new ApprenticeshipLocation()
+                {
+                    Address = l.Address,
+                    ApprenticeshipLocationType = l.ApprenticeshipLocationType,
+                    CreatedBy = request.CreatedByUser.Email,
+                    CreatedDate = request.CreatedDate,
+                    DeliveryModes = EnumHelper.SplitFlags(l.ApprenticeshipLocationType).Cast<int>().ToList(),
+                    Id = Guid.NewGuid(),
+                    LocationType = l.LocationType,
+                    Name = l.Name,
+                    National = l.National,
+                    Phone = l.Phone,
+                    ProviderUKPRN = request.ProviderUkprn,
+                    Radius = l.Radius,
+                    RecordStatus = 1,
+                    Regions = l.Regions,
+                    UpdatedBy = request.CreatedByUser.Email,
+                    UpdatedDate = request.CreatedDate,
+                    VenueId = l.VenueId
+                }).ToList(),
+                RecordStatus = 1,
+                CreatedDate = request.CreatedDate,
```

Comment:
These two `Created` fields shouldn't be overwritten (they shouldn't really be part of command at all)

Reply:
removed these now, and updated the test handler too.